macros/unpack/get_source_values.sql

```diff
@@ -29,6 +31,8 @@
               wrap_string_with_quotes(node.package_name),
               wrap_string_with_quotes(node.loader),
               wrap_string_with_quotes(node.identifier),
+              wrap_string_with_quotes(node.tags | tojson),
```

Comment:
@alexfetterman i think the `tojson` from jinja may not handle the structure of the data super well. Can you try syntax like [this field](https://github.com/dbt-labs/dbt-project-evaluator/blob/main/macros/unpack/get_metric_values.sql#L23) in our metrics macro? i think this may work better